Code/Engine/Foundation/Communication/Implementation/IpcChannel.cpp

```diff
@@ -174,7 +174,7 @@ void ezIpcChannel::ReceiveMessageData(ezArrayPtr<const ezUInt8> data)
 
       ezProcessMessage* pMsg = (ezProcessMessage*)ezReflectionSerializer::ReadObjectFromBinary(reader, pRtti);
       ezUniquePtr<ezProcessMessage> msg(pMsg, ezFoundation::GetDefaultAllocator());
-      if (msg != NULL)
+      if (msg.Borrow() != nullptr)
```

Comment:
Borrow() shouldn't be necessary since ezUniquePtr has comparison operators for nullptr